src/shared/media-types.ts

```diff
@@ -1,7 +1,8 @@
 enum MediaTypes {
   MOBILE = '576',
-  TABLET_M = '768',
-  TABLET_L = '1024',
+  TABLET = '768',
```

Comment:
updated TABLET media types naming for 2 reasons:
1) `992px` was missing
2) tabletMMax is not very comfortable to use